src/common/armor.cc

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Liu Changcheng <changcheng.liu@aliyun.com>
```

Comment:
are you sure about this? if you create this PR at work, and since, normally the copyright of work for hire belongs to the employer, the copyright declaration is likely to be wrong.

Reply:
@tchaikov 
I'm not sure about this.
I'll follow your suggestion. I'll refer to https://github.com/ceph/ceph/blob/master/src/common/EventTrace.h#L6 to add the license header.
Which license do you prefer? Apache or BSD or GPL2.1

Reply:
Done